imsv-docs-astro/src/content/docs/guides/core-concepts/card-funding.mdoc

```diff
@@ -42,41 +42,60 @@ of digital assets needed to meet a users desired spend in their desired fiat
 currency. They also provide pre-built `"smart-contract-write"` transactions for
 successful interactions with the Immersve smart contract.
 
+If multiple transactions are present then they should be carried out in the
+order in which they are presented.
 
-### Currency Conversion
+The client application is responsible for creating the raw blockchain transaction
+so that it can be signed and submitted to the relevant blockchain through the
+cardholder's web3 wallet.
 
-A user may be quoted a price for a purchase by a merchant in a local fiat
-currency. To determine the sufficient amount of local fiat currency to fund a
-card in its billing currency (USD), use the [currency
-conversion](/api-reference/currency-conversion) API endpoint.
+<!-- https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764577778509789&cot=14 -->
+![Card Funding Sequence](@assets/diagrams/card-funding-sequence.svg)
 
-The returned value can passed to the get spending prerequisites endpoint as
-detailed below.
 
+## Withdrawing Funds
 
-### Lock Funds
+Withdrawals are conducted as a two-stage process being;
```

Comment:
```suggestion
Withdrawals are conducted as a two-stage process:
```
This line reads a bit odd to me, do we need "being" at the end?

Reply:
Thanks Kira, fixed!